### ISSUE TITLE
Can the PinQR Helper webpage be included in this repo too?

### ISSUE THREAD

**Issue by 3rdIteration** [Blockstream/blind_pin_server#16]
I am playing around with self-hosted PIN servers and noticed that if you follow the currently available documentation and set it up without HTTPS, then you can't use the PinQR helper website hosted by blockstream. 

Simply downloading the helper and running it on a webhost over http works just fine. (Though none of the source includes any license information, so I'm loathed to simply add that and bring it via a PR)

I'll likely also just include the settings needed to run the PIN server via HTTPS in a video that I will be making on this matter. Basically this would be helpful for those who want to have a fully self-hosted setup and still log in via PinQR.

**Comment by m-schmoock** [comment]
Yes, that would be helpful. Not necessarily the Blockstream style page, but maybe a simple page without style, just showing the QRs would be sufficient.

**Comment by petre-c** [comment]
@3rdIteration , have you managed to make the video?
I'm unable to show the QR code and was wondering how to fix this..

**Comment by 3rdIteration** [comment]
They (blockstream) are doing some work on the pinserver for something like Umbrel and this will likely include a docco refresh, so I am basically waiting to see what they end up releasing with that.

Are you asking about how to self host the PinQR helper or how to use the pinserver generally?

**Comment by petre-c** [comment]
> They (blockstream) are doing some work on the pinserver for something like Umbrel and this will likely include a docco refresh, so I am basically waiting to see what they end up releasing with that.
> 
> Are you asking about how to self host the PinQR helper or how to use the pinserver generally?

I followed the official guides on this repo and for Jade and managed to set up and launch the pinserver, albeit with many hickups along the way.
Also changed the pinserver url in Jade hardware wallet - specified 
```python
--set-url https://127.0.0.1:8069
```
instead of the url address without the scheme.

pinserver is started at http://127.0.0.1:8069 but the index pages is returning empty body - I don't even know what it should look like or whether I should expecting the similar browser-based workflow as the one officially supported through their website.

**Comment by 3rdIteration** [comment]
Basically, unless you are self-hosting the PinQR helper, it needs to be running on the pinserver with enabled HTTPS with a valid certificate, an SSL cert from Letsencrypt works fine, or a self-signed cert that your browser trusts. (If you self host the PinQR helper then you can get away with running everything over http)

**Comment by petre-c** [comment]
I will start by selfhosting the PinQR helper and will likely end there, I'm just trying to set up the PinQR for a fully offline infrastructure and workflow for myself.

**Comment by petre-c** [comment]
> Simply downloading the helper and running it locally over http works just fine

Where does one get the PinQR helper from?

**Comment by 3rdIteration** [comment]
The simple answer is that you don't, that is what this issue is asking for. (I have a working copy, but it isn't licenced in a such a way as to allow it to be shared)

That said, it is possible to download it manually from the PinQR website if you know how. (And if you don't, the best solution is to get HTTPS working properly on your blind pinserver)

**Comment by petre-c** [comment]
Okay, thank you!

**Comment by petre-c** [comment]
@JamieDriver 
@greenaddress 
@nitramiz 

please answer

**Comment by JamieDriver** [comment]
@bitcoinhelp

**Comment by bitcoinhelp** [comment]
https://github.com/Blockstream/esp32_bc-ur/tree/emscripten

https://github.com/Blockstream/esp32_bc-ur/blob/emscripten/EMSCRIPTEN.md (small typo it looks like, http://127.0.0.1:9990 should of course be http://127.0.0.1:9999)

**Comment by petre-c** [comment]
You may want to look at an alternative implementation of the blind oracle / pinserver here -> https://github.com/Filiprogrammer/SimpleJadePinServer
